schema/trade.proto

```diff
@@ -557,6 +595,14 @@ message Fill {
   uint64 subaccount_id = 9;
   // The cumulative filled quantity for this order after the fill is applied.
   uint64 cumulative_quantity = 10;
+  // A multiplicative factor that indicates the fee charged on this trade
+  // when combined with with `fee_ratio_exponent``.
+  // See [Fees](#fees) for details.
+  int64 fee_ratio_mantissa = 11;
+  // An exponential factor that indicates the fee charged on this trade
+  // when combined with with `fee_ratio_mantissa`.
+  // See [Fees](#fees) for details.
+  int64 fee_ratio_exponent = 12;
```

Comment:
Would not it be better to represent fee_ration as a compound value?  
Until https://github.com/protocolbuffers/protobuf/issues/4406 is resolved, we could define our own type Decimal = {Marissa, exponent}. 
Or just simplify that and say that our fees_ration is always in pips.

Reply:
Would you clarify what you mean by "compound value" in this context?

FIX has a preferred format for fixed-point decimals and this is it.  Doing it the same way here to match.  I don't love it.  It's definitely confusing but at least it's precise.

It would be great to say "fees are in pips" but there's a possibility that we might want fee granularity of <1 pip in the future - adding the exponent field later would require changing every client that hardcoded the exponent, which would probably annoy people more.

Reply:
What I mean is that we define the message as: 
CubeDecimal fee_ratio;

and then
```
message CubeDecimal {
  int64 mantissa =1;
  int64 exponent=2;
}
```

Or we just scrap that and make just a field: 
int64 fee_ratio_pips. 

If we want to increase precision, we'll just add another field: 
int64 fee_ration_milli_pips;

We would still fill both fields on the server. 
This will not e a breaking change and old clients will continue working. Old clients would just see fee_ration_pip as 0 since it will be rounded down.

Reply:
this is added